packages/rest/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:18.04 as base
```

Comment:
I think you wanted me to do this once and I kinda forgot, but don't we want to create an afj image with libindy and all installed?

Reply:
Yeah may be still worth looking at.

(here's the issue: https://github.com/animo/indy-nodejs-docker/issues/1)